### PR TITLE
[Prerelease] Upgrade to Angular v10

### PR DESCRIPTION
**Pull request by gund**
Now the library is built using Angular v10.
No public API changes.

### CODE REVIEW

**Comment by gund** [comment]
CI is failing on the branch job - https://travis-ci.org/github/orchestratora/orchestrator/builds/703010902#L529
It requires proper initialization of git so that master ref can be used by NX CLI (like in the PR job).

File: decorate-angular-cli.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+/**
```
Comment (RyuuGan) [inline decorate-angular-cli.js:1]: Is this file autogenerated ?

File: libs/core/package.json [hunk 2]
```diff
@@ -19,13 +19,13 @@
   "license": "MIT",
   "peerDependencies": {
     "@angular/common": "^9.0.0",
-    "@angular/core": "^9.0.0",
-    "tslib": "^1.0.0"
+    "@angular/core": "^9.0.0"
```
Comment (RyuuGan) [inline libs/core/package.json:22]: If it is migrating to Angular 10, why it is still `@angular/core` version 9 ?

File: libs/layout/package.json [hunk 3]
```diff
@@ -21,7 +21,9 @@
     "@angular/cdk": "^9.0.0",
```
Comment (RyuuGan) [inline libs/layout/package.json:21]: Why here are `@angular` modules version 9 ?

File: libs/stepper/package.json [hunk 4]
```diff
@@ -20,7 +20,9 @@
   "peerDependencies": {
     "@angular/common": "^9.0.0",
```
Comment (RyuuGan) [inline libs/stepper/package.json:21]: Why here are `@angular` modules version 9 ?

File: libs/ui-web/package.json [hunk 5]
```diff
@@ -21,7 +21,9 @@
   "peerDependencies": {
     "@angular/common": "^9.0.0",
```
Comment (RyuuGan) [inline libs/ui-web/package.json:22]: Why here are `@angular` modules version 9 ?

File: decorate-angular-cli.js (hunk 1 above)
Comment (gund) [reply decorate-angular-cli.js:1]: Yep, this was added by the NX CLI migrations.

File: libs/core/package.json (hunk 2 above)
Comment (gund) [reply libs/core/package.json:22]: I have these versions updated locally, just did not commit yet as I'm trying to figure out the errors in Travis CI...

**Comment by gund** [comment]
Travis CI issue is resolved!

Turns out that after NX upgrade some commands (`format:check`) require local master branch to be present in git.
However Travis CI does not fetch anything except the branch in the build and git will see only one branch - which causes fails in the tooling.

To fix this issue one needs to reset git to fetch all branches:
```
$ git config --replace-all remote.origin.fetch +refs/heads/*:refs/remotes/origin/*
```

And then create local remote tracking branch for master:
```
$ git fetch origin master
$ git branch -tf master origin/master
```

File: libs/core/package.json (hunk 2 above)
Comment (gund) [reply libs/core/package.json]: Package updated to v10.

File: libs/stepper/package.json (hunk 4 above)
Comment (gund) [reply libs/stepper/package.json]: Package updated to v10.

File: libs/ui-web/package.json (hunk 5 above)
Comment (gund) [reply libs/ui-web/package.json]: Package updated to v10.

File: libs/layout/package.json (hunk 3 above)
Comment (gund) [reply libs/layout/package.json]: Package updated to v10.

**Pull request merged**